pages/common/texcount.md

```diff
@@ -0,0 +1,20 @@
+# texcount
+
+> Count words in TeX documents omiting macros.
+> More information: <https://app.uio.no/ifi/texcount/index.html>.
+
+- Count words in one TeX file:
+
+`texcount path/to/file`
+
+- Count words in document and subdocuments built with \input or \include:
+
+`texcount -merge file`
+
+- Count words in document and subdocuments list each file extra:
+
+`texcount -inc file`
```

Comment:
```suggestion
- Count words in a document and subdocuments, listing each file separately (and a total count):

`texcount -inc {{path/to/file.tex}}`
```

Reply:
So the problem is that when the program `texcount -merge` or `texcount -inc` is called and the root TeX file includes or inputs other files, the program call needs to be from the root TeX file directory. Otherwise it doesn't find the linked files. At least when I tested it this behaviour was the case.

Any idea how to make this clear?